vendor/bam/tests/place_android_phone_order_request.json

```diff
@@ -3,10 +3,10 @@
   "id": 10,
   "payload": {
     "type": "PLACE-ORDER",
-    "headers" : {
-      "PRODUCT-TYPE" : "PHONE"
+    "headers": {
+      "PRODUCT-TYPE": "PHONE"
     },
-    "body" : {
+    "body": {
```

Comment:
🤡 I'm guessing this wasn't intentional. Let's keep prettify to api_tests only 😅.

Reply:
Hum no, it's intentional. Prettify all Json and JS files.

Reply:
ok, removed...